### DISCUSSION TITLE
@EnableLoginArgResolver注解开源版本找不到

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#214] [02.提问] [answered]
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/6092525/199447144-d3571898-0a1c-464c-a8cf-34a51653f86c.png">

**Comment by zuihou** [comment, marked as answer]
没有了

LoginArgResolverConfig 启动时会自动加载，无需加这个注解了
